README.md

```diff
@@ -204,6 +214,6 @@ Sometimes, you may want to make your extension configurable by the extension end
 |:----------------:|:------------------------------:|
 | 2.3.0 - current  | 0.3.0                          |
 
-| jib-gradle-plugin | jib-gradle-plugin-extension-api |
-|:-----------------:|:-------------------------------:|
-| 2.4.0 - current   | 0.3.0                           |
+| jib-gradle-plugin | jib-gradle-plugin-extension-api | Jib Plugin Runtime Gradle API |
```

Comment:
I don't know if people will understand what jib plugin runtime gradle api means? maybe a `*` and a description somewhere?